website/blog/2023-08-23-silver.md

```diff
@@ -0,0 +1,372 @@
+---
+
+We're happy to announce the release of Metals v1.0.1, which comes with
+a couple of exciting new features and stability improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">72</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">66</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/59?closed=1]
+(https://github.com/scalameta/metals/milestone/59?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+- [Completions in java files](#completions-in-java-files)
+- [Liveness monitor](#liveness-monitor)
+- [Improvements for enums](#improvements-for-enums)
+- [Semantic highlighting](#semantic-highlighting)
+- [Miscellaneous](#miscellaneous)
+
+
+## Completions in Java files
+
+Thanks to amazing work done by [VankaTaganai](https://github.com/VankaTaganai) and further improvements by [tgodzik](https://github.com/tgodzik),
+you will be able to get completions in Java files. It should make 
+working with Java sources easier, without striving to be a full Java language server.
+
+<< add java completion gif >>
+
+## Liveness monitor
+
+Up to the current release, users would not be informed if something happened,
+and the build server stopped responding. Thanks to [kasiaMarek](https://github.com/kasiaMarek) Metals will now monitor the state 
+of the build server and notify the user if anything goes wrong.
+
+<< add notification img >>
+
+## Improvements for enums
+
+This release also bring in multiple fixes connected to enums: 
+
+- better highlighting on enum cases
+- improved navigation on enum parameters
+- multiple fixes on enum type parameters
+
+This great work was done by [kasiaMarek](https://github.com/kasiaMarek)
+
+<< add gif with enum highlight >>
+
+## Semantic highlighting
+
+Semantic highlighting enables syntax highlighting based on knowledge from the compiler.
+Thanks to this, users can have specific colors for parameters, methods, mutable `var`s, packages etc. 
+For the last half year, semantic highlighting was hidden behind a flag, but now we think it's ready 
+to be turned on by default. You can always change this using `metals.enableSemanticHighlighting` setting. 
+Some editors also require enabling this feature other places. 
+In VS Code, you also need to set `editor.semanticHighlighting.enabled` to `true`.
```

Comment:
Maybe link to where it was first announced? We have the settings there

Reply:
Good idea 👍